hatchways/team-rambo
# SendGrid Email API Integration

Integrate SendGrid's Email API to send reminder emails for Board deadlines

### What this PR does (required):
- Integrate SendGrid API for future emails of board deadlines to users.
- Using temporary values for `to/from` emails until infrastructure is decided by the team.

### Screenshots / Videos (required):
- Video [here](https://www.loom.com/share/32b2c8df9ba94363a207ecaeaa2e78ad) in action

### Any information needed to test this feature (required):
- Ensure you are logged out as a user.
- Open the server console and see the `Email Sent` message.
- Currently, have it setup to send directly to my email from an email I created for this feature; to edit for testing, go to `sendEmail.js` and substitute my email in the `to:` to yours

### Any issues with the current functionality (optional):
- Will need to adapt this feature to the correct method when we want to implement updates for users' board status
